tests/unit/test_schemas.py

```diff
@@ -26,13 +26,30 @@ def stories_schema():
     return YAML(typ="safe").load(open(schema_file, encoding="utf-8"))
 
 
+@pytest.fixture
+def plans_schema():
+    # TODO: tmt package shall provide a helper function
+    # for "load schemas for test/plan/story"
+    schema_file = os.path.join(SCHEMADIR, 'plan.yaml')
+    return YAML(typ="safe").load(open(schema_file, encoding="utf-8"))
+
+
 @pytest.fixture
 def schema_store():
     # TODO: tmt package shall provide a helper function
     # for "load schemas for test/plan/story"
     store = {}
 
-    for schema_name in ('common', 'core'):
+    for schema_name in (
+            'common', 'core',
+            'discover/fmf', 'discover/shell',
+            'execute/tmt', 'execute/upgrade',
+            'finish/ansible', 'finish/shell',
+            'prepare/ansible', 'prepare/install', 'prepare/shell',
+            'provision/artemis', 'provision/connect', 'provision/container',
```

Comment:
Missing `provision/hardware`. The file should be renamed from `hardware.yml` to `hardware.yaml`.

Reply:
(FYI, https://github.com/teemtee/tmt/pull/1266 should take care of the "we forgot to list a file" issue)

Reply:
@psss thanks, added and renamed